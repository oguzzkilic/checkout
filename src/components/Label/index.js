import styled from 'styled-components';

const Label = styled.label`
  font-family: Arial;
  font-size: 14px;
  padding-bottom: 10px;
  display: block;
  color: ${props => props.theme.inputTextColor};
`;

export default Label;
