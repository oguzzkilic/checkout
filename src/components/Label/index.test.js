import React from 'react';
import Renderer from 'react-test-renderer';
import Label from './index';

import 'jest-styled-components';

describe('Label Component Tests', () => {
  it('renders correctly', () => {
    const tree = Renderer
      .create(<Label>Message from world</Label>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
