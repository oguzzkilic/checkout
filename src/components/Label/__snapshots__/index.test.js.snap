// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Component Tests renders correctly 1`] = `
.c0 {
  font-family: Arial;
  font-size: 14px;
  padding-bottom: 10px;
  display: block;
}

<label
  className="c0"
>
  Message from world
</label>
`;
